classes/sdk-relocate-toolchain.bbclass

```diff
@@ -1,6 +1,7 @@
 SDKPATHTOOLCHAIN ?= "${SDKPATH}/toolchain"
 
 SDK_POST_INSTALL_COMMAND:append() {
+
```

Comment:
Tricky. Looking at the commit message I cannot say what's broken if we don't do this.